news/5345.behavior.rst

```diff
@@ -0,0 +1 @@
+New pipfiles show python_full_version under [requires] if specified
```

Comment:
Can you please add a longer explanation here?

Reply:
ok cool. good call. Hows it now?

Reply:
Much better. Thank you.